src/lib/features/listening-history/model/specs/store.spec.ts

```diff
@@ -13,7 +13,7 @@ const exampleEpisode: Episode = {
 
 const expectedHistory: number[] = [exampleEpisode.id];
 
-it('listen to an episode', () => {
+it('add episode to the history', () => {
```

Comment:
nitpick (non-blocking): I like to think of this function name `it()` as a part of the sentence, so to be grammatically correct, we would say "it adds an episode..."
```suggestion
it('adds an episode to the history', () => {
```